### PR TITLE
Css file for Dufuna.html 

### PR DESCRIPTION
**Pull request by abolayo**
i added registration form dufuna.html
i added dufuna.css for the dufuna.html file
i added an image file for logo

### CODE REVIEW

**Review by yemexx1 — DISMISSED** [review]

File: submissions/medinatapanpa/html/dufuna.html [hunk 1]
```diff
@@ -2,23 +2,23 @@
...
  thier finest flavours.</article>
-<style>
-  form { align-self: auto;
-    }
-</style>
- <P>
+
+ <P>  
+   <div class="bigbox">
     <form style="align-content: space-between;">
```
Comment (Ifeoluwa) [inline submissions/medinatapanpa/html/dufuna.html]: Please move inline style definition to your stylesheet for easy maintainability.

Comment (abolayo) [reply submissions/medinatapanpa/html/dufuna.html]: I have done that. thank you

**Review by adaugochi — APPROVED** [review]

File: submissions/medinatapanpa/html/dufuna.html [hunk 2]
```diff
@@ -2,23 +2,23 @@
...
- thier finest flavours.</article>
-<style>
-  form { align-self: auto;
-    }
-</style>
- <P>
-    <form style="align-content: space-between;">
+ their finest flavours.</article>
+
+ <P>
```
Comment (kunmi02) [inline submissions/medinatapanpa/html/dufuna.html:16]: Your tags do not follow a pattern. Please stick to lowercase

File: submissions/medinatapanpa/html/dufuna.html [hunk 3]
```diff
@@ -36,16 +36,14 @@
...
       
        </fieldset>
-        
-        <div class="container signin">
-          <p>Already have an account? <a href="#">Sign in</a></p>
+      
+        <div class="signin">
+          <p>Already have an account?</p> <button class="button2" a href="#">Sign in</a> </button>
         </div>
       </form>
```
Comment (kunmi02) [inline submissions/medinatapanpa/html/dufuna.html:46]: check your identations

**Review by kunmi02 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**